detox/src/devices/Device.js

```diff
@@ -180,6 +181,19 @@ class Device {
     await this.deviceDriver.uninstallApp(this._deviceId, _bundleId);
   }
 
+  async installUtilBinaries() {
+    const paths = this._deviceConfig.utilBinaryPaths;
+    if (paths) {
+      if (!_.isArray(paths)) {
+        throw new DetoxRuntimeError({
```

Comment:
@d4vidi, I think this is very far from its origination. What if we move that validation inside `detox/configuration/*` layer?

Reply:
Yes - will revisit!

Reply:
updated! see https://github.com/wix/Detox/pull/2188/commits/936e164e3822f09d005d3bf7a48316fef98add0c

Reply:
I see, but let's refine how the error will be looking on the user screen. Should be neutral, terse, and user-friendly.

Reply:
ok, will apply all your suggested fixes asap 🙏

Reply:
please revisit 😄 https://github.com/wix/Detox/pull/2188/commits/d875883b843e85079406d8bbdb2f5fb4b37d3c4a